code/obj/cryotron.dm

```diff
@@ -361,9 +342,65 @@
 		if (!exit_prompt(user))
 			return ..()
 
+	/// Handling dragging players in to cryo, mainly for silicon players.
 	MouseDrop_T(atom/target, mob/user as mob)
-		if (ishuman(target) && isrobot(user) && BOUNDS_DIST(src, user) == 0 && BOUNDS_DIST(src, target) == 0 && BOUNDS_DIST(user, target) == 0)
-			insert_prompt(target, user)
+		if (!ishuman(target) && !isrobot(target))
```

Comment:
not the right logic

Reply:
I believe I have fixed this now, would appreciate another look when you have the chance, appologies for the delay I haven't been able to get back to this for a while.